manualconfig/routertopology/fanout/readme.md

```diff
@@ -0,0 +1,237 @@
+
+# Configuring AMQ Interconnect router in fan-out topology
+
+* [Router configs](#router-configs)
+  + [Router 1 config](#router-1-config)
+  + [Router 2 config](#router-2-config)
+  + [Router 3 config](#router-3-config)
+* [Firewall settings](#firewall-settings)
+* [Run the routers](#run-the-routers)
+* [Prerequisite](#prerequisite)
+
+In the fan-out topology, both producers and consumers will be connecting to the main (hub) router
+which will fan-out the messages to two routers (spokes). Each of these two routers will be connected
+to a live AMQ broker.
+
+In this architecture, each of the component (router/broker) will be running on a separate VM.
+
+_The configuration file for each of the router is_ `/etc/qpid-dispatch/qdrouterd.conf`.
+
+
+## Router configs
+
+### Router 1 config
+This is the router which will serve producers and consumers. This router provides a 
+few listeners to support incoming connections from consumer/producers as well
+as other routers in the network.
+
+* Change the `router` element in the config file to:
+```shell
+router {
+    mode: interior
+    id: router.fan-out-hub-01
+}
+```
+
+* Add the config given below at the end of the config file
+
+
+<details>
+  <summary>Click to expand or collapse the config</summary>
+
+```shell
+# Listener for inter-router comm
+listener {
```

Comment:
Listener for client connections - for accepting connections from producer/consumers